content/en/v3/admin/platforms/k3s/_index.md

```diff
@@ -19,8 +19,6 @@ Ensure you are logged into GitHub else you will get a 404 error when clicking th
 
```

Comment:
Let's remove these changes from this PR, and publish the blog. We can address the issues with k3s installation in a separate PR, wdyt?

Reply:
Of course. Just me and git not being friends 😝